terraform/validation/main.tf

```
@@ -73,7 +73,7 @@ resource "null_resource" "validator" {
   provisioner "local-exec" {
     command = <<-EOT
       docker-compose -f ${local.docker_compose_path} down
-      docker-compose -f ${local.docker_compose_path} build
+      docker-compose -f ${local.docker_compose_path} build --no-cache
```

```suggestion
      docker-compose -f ${local.docker_compose_path} build
```

This was only needed for local builds. We can use the cache normally.